### DISCUSSION TITLE
Show priorities

### DISCUSSION THREAD

**Discussion by jbtodo** [ransome1/sleek#736] [General]
Is there a way to display priorities in the app list view?
I see that A - C have colors but I would like to display the priority letter in all of the tasks, rather than need to open them to see what priority each is. 
Or am I missing something?
Thanks!

**Comment by docnux** [comment]
The tasks are compiled under the corresponding priority letter. For me this is sufficient.
... or are your task lists so long that the priority letters get out of view?

**Reply by jbtodo** [reply]
They are in order, that was not what I was suggesting.
Currently the tasks show Pri B - Pri D. Any item tagged Pri A or E + is in order but you cannot see the priority without opening it.
I am asking to to display ALL of the priority tags.
